configuration/lambda_config.json

```diff
@@ -5,5 +5,35 @@
   },
   "headers": {
     "max_age": "600"
+  },
+  "logging": {
+    "version": 1,
+    "formatters": {
+      "brief": {
+        "format": "%(asctime)s - %(levelname)s - %(message)s",
```

Comment:
Is this throwing away some useful info compared to the default handler?  That handler seems to somehow be able to add the lambda request/event ID into the log messages, we'd better keep that if possible.